src/OSPSuite.Core/Comparison/StartValueDiffBuilder.cs

```diff
@@ -30,7 +31,9 @@ protected virtual void ComparePathAndEntityValues(IComparison<T> comparison)
          // of a distributed PathAndValueEntity is not fully calculated until  it is turned into
          // a parameter
          if(!comparison.Object1.DistributionType.Equals(comparison.Object2.DistributionType))
-            CompareValues(x => x.DistributionType, x => x.DistributionType, comparison);
+            CompareValues(x => x.DistributionType, x => x.DistributionType, comparison, 
+               areEquals: (type1, type2) => Equals(type2, type1), 
```

Comment:
should we rename the argument _areEqual**s**_ to _areEqual_?

Reply:
yes, done